pages/hold/request/[id]/edd.tsx

```diff
@@ -250,6 +253,10 @@ export async function getServerSideProps({ params, req, res }) {
 
     const patronEligibilityStatus = await fetchPatronEligibility(patronId)
 
+    // fetch patron's email to pre-populate the edd form if available
+    const patronData = await getPatronData(patronId)
```

Comment:
this is fetching a lot of patron data, though it doesn't appear to be taking a very long time. I still think that instead of using `getPatronData`, which builds out a whole my account dataset including checkouts and holds and fines, maybe you can use the `MyAccount` model's `getPatron`

Reply:
Don't I still need to initialize a MyAccount data model to use the getPatron function?

Reply:
We talked offline and I ended up building a MyAccount model instead of calling getPatron() since this counterintuitively involves less data instantiation.